### ISSUE TITLE
A wordlist is required but a default could not be found.

### ISSUE THREAD

**Issue by zhangxiaoshang** [bitcoinjs/bip39#115]
step1
```bash
browserify -r bip39 -s bip39 > bip39.browser.js
```

step2
```html
<!DOCTYPE html>
<html lang="cn">
<head>
    <meta charset="UTF-8">
    <title>Document</title>
    <script src="./bip39.browser.js"></script>
    <script>
    console.log('bip39.wordlists', bip39.wordlists) // {}
    var mnemonic = bip39.generateMnemonic()
    console.log('mnemonic', mnemonic) // not work
    </script>
</head>
<body>
    
</body>
</html>
```

The error message:
 ```bash
bip39.wordlists {}
20:31:17.213 bip39.browser.js:6009 Uncaught Error: A wordlist is required but a default could not be found.
Please explicitly pass a 2048 word array explicitly.
    at entropyToMnemonic (bip39.browser.js:6009)
    at Object.generateMnemonic (bip39.browser.js:6036)
    at (index):9
entropyToMnemonic @ bip39.browser.js:6009
generateMnemonic @ bip39.browser.js:6036
(anonymous) @ (index):9
```

(infact i want create mnemonic with react-native and expo)Please help~

**Comment by junderw** [comment]
please provide an example project that I can run npm install and npm start etc.

Then I can see what problems you're having.

Your code is insufficient to reproduce the issue.

**Comment by junderw** [comment]
Also:

1. node -v
2. npm -v
3. browserify --version

**Comment by junderw** [comment]
#116 fixes the problem.

I published patch 3.0.2... so please try reinstalling bip39 and then try again.
